### DISCUSSION TITLE
Use different hostnames for webapp & mail server

### DISCUSSION THREAD

**Discussion by somedudeokey** [Mailu/Mailu#3298] [User support]
Hello

I have added certificates to "/mailu/certs" which works fine so far. I can visit the https version of webadmin and webmail. What I'm wondering is if mailu uses the same certificates to send and receive emails?

For example, I want the web application to be reached through 'webmail.domain.com'

and use only 'domain.com' as hostname for the mail server

I assume this requires two different certificates, one for webmail.domain.com and one for domain.com

**Comment by nextgens** [comment]
Add a new hostname to your mailu.env file (look for ``HOSTNAMES``) and recreate your front container
